etc/kayobe/environments/ci-aio/automated-setup.sh

```diff
@@ -7,6 +7,7 @@ cat << EOF | sudo tee -a /etc/hosts
 EOF
 
 if sudo vgdisplay | grep -q lvm2; then
```

Comment:
I wonder why this is necessary when running this script, but not in CI.

Reply:
Different images? We're using the Ubuntu-20.04 image but CI uses overcloud-ubuntu-focal-*

Reply:
I've also seen it do different things when running things as different users in the past. Maybe it's something about when we run things as `ubuntu` vs `root` for example

Reply:
We should switch to use the release train images.

Reply:
Running through an aio on a CS8 release train image now. This part is required because we clone repos and create virtualenvs in /home, whereas in CI that's done on the CI runner.